.github/contributing.md

```diff
@@ -1,3 +1,160 @@
-# How to contribute
+# UGRC website contributing guide
 
-Read the following http://gis.utah.gov/about/contributing/
+Thank you for your interest in contributing to the UGRC website. Before you proceed, briefly go through the following:
+
+- [Code of Conduct](https://github.com/agrc/gis.utah.gov/blob/HEAD/CODE_OF_CONDUCT.md)
+- [Contributing](#contributing)
+- [Getting started](#getting-started)
+  - [CLI Commands](#cli-commands)
+- [Commit Guidelines](#commit-guidelines)
+- [Pull Request Policy](#pull-request-policy)
+
+## Contributing
+
+Any individual is welcome to contribute to the UGRC website.
+
+## Getting started
+
+The steps below will give you a general idea of how to prepare your local environment for the UGRC website and general steps
+for getting things done and landing your contribution.
+
+1. Click the fork button in the top right to clone the [UGRC website Repository](https://github.com/agrc/gis.utah.gov/fork)
+
+2. Clone your fork using SSH, GitHub CLI, or HTTPS.
+
+   ```bash
+   git clone git@github.com:<YOUR_GITHUB_USERNAME>/gis.utah.gov.git # SSH
+   git clone https://github.com/<YOUR_GITHUB_USERNAME>/gis.utah.gov.git # HTTPS
+   gh repo clone <YOUR_GITHUB_USERNAME>/gis.utah.gov # GitHub CLI
+   ```
+
+3. Change into the gis.utah.gov directory.
+
+   ```bash
+   cd gis.utah.gov
+   ```
+
+4. Create a remote to keep your fork and local clone up-to-date.
+
+   ```bash
+   git remote add upstream git@github.com:agrc/gis.utah.gov.git # SSH
+   git remote add upstream https://github.com/agrc/gis.utah.gov.git # HTTPS
+   gh repo sync agrc/gis.utah.gov # GitHub CLI
+   ```
+
+5. Create a new branch for your work.
+
+   ```bash
+   git checkout -b name-of-your-branch
+   ```
+
+6. Run the following to install the dependencies and start a local preview of your work.
+
+   ```bash
+   npm ci # installs this project's dependencies
+   npm start # starts a preview of your local changes
+   ```
+
+7. Perform your changes. In case you're unfamiliar with the structure of this repository, we recommend a read on the [Collaborator Guide](./collaborating.md)
```

Comment:
```suggestion
7. Perform your changes. In case you're unfamiliar with the structure of this repository, we recommend that you review the [Collaborator Guide](.github/collaborating.md).
```